src/script/components/Modals/GroupCreation/GroupCreationModal.tsx

```diff
@@ -53,6 +54,7 @@ import {
 import useEffectRef from 'Util/useEffectRef';
 import {useFadingScrollbar} from '../../../ui/fadingScrollbar';
 import {Config} from '../../../Config';
+import {theme} from '../../../page/MainContent/MainContent';
```

Comment:
I'm making a note of this. The theme is going to live in the ui-kit soon. I'll make sure to update this

Reply:
Sounds good, thanks!